app/views/publish/courses/deletions/edit.html.erb

```diff
@@ -0,0 +1,52 @@
+<% content_for :page_title, title_with_error_prefix("Are you sure you want to withdraw #{@course.name_and_code})?", @course_deletion_form.errors.any?) %>
+
+<% content_for :before_content do %>
+  <%= govuk_back_link_to(publish_provider_recruitment_cycle_course_path(
+    @course.provider_code, 
+    @course.recruitment_cycle_year, 
+    @course.course_code)
+  ) %>
+<% end %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <%= form_with(
+      model: @course_deletion_form,
+      url: delete_publish_provider_recruitment_cycle_course_path(@provider.provider_code, @course.recruitment_cycle_year, @course.course_code),
+      data: { qa: "enrichment-form", module: "form-check-leave" },
+      method: :delete,
+    ) do |f| %>
+
+      <%= f.govuk_error_summary %>
+
+      <h1 class="govuk-heading-l">
+        <span class="govuk-caption-l"><%= @course.name_and_code %></span>
+        Are you sure you want to delete this course?
+      </h1>
+
+    <p class="govuk-body">You can only delete a course if it has not been published in this cycle.</p>
```

Comment:
should we lockdown the controller for showing a published course?

Reply:
good call

Reply:
done https://github.com/DFE-Digital/teacher-training-api/pull/2579/commits/264ad5cbe3316a49ffecd06367945b3d7f61ebe0